scripts/install-multi-user.sh

```diff
@@ -387,19 +387,26 @@ EOF
     fi
 
     for profile_target in "${PROFILE_TARGETS[@]}"; do
+        # TODO: I think it would be good to accumulate a list of all
+        #       of the copies so that people don't hit this 2 or 3x in
+        #       a row for different files.
         if [ -e "$profile_target$PROFILE_BACKUP_SUFFIX" ]; then
             failure <<EOF
-When this script runs, it backs up the current $profile_target to
-$profile_target$PROFILE_BACKUP_SUFFIX. This backup file already exists, though.
+Starting with Nix 2.1, I back up shell profile/rc scripts before I add
```

Comment:
Is the "Starting with Nix 2.1" bit necessary?

Reply:
I'm thinking ahead about something, but I suppose not it's not adding anything for now. 

I'll move that into a comment in a force-push, after I get feedback from toonn.